round4/group16

```diff
@@ -0,0 +1,60 @@
+Chrome Extension Idea: AI Research Assistant
+Authors
+Runzhi Gu, Sam Bhatta, Daniel Kwon, Caroline Begg
+
+Problem Statement
+Students, researchers, and information-seekers often struggle with the overwhelming task of sifting through vast amounts of online content to find relevant information. There is a lack of effective tools that are embedded in the web browser that helps the users quickly understand a long and complicated web article, which makes the research process daunting and less productive.
+
+Target Audience
+Students conducting research for academic papers or projects. Researchers gathering up to date information across various fields. Educators and content creators that collect rephrased information. Anyone who frequently engages in extensive online reading and needs to compile information efficiently. 
+
+Description
+The AI Research Assistant is a Chrome Extension powered by Open AI api through the GPT model, designed to enhance the online research process. It offers a sidebar interface that allows users to summarize web pages or specific sections, and displays it in an organized and refined manner. 
+
+Selling Points
+Efficient Information Processing: The extension swiftly summarizes long and complex web articles, condensing them into digestible highlights. This drastically reduces the time required for users to sift through and understand vast amounts of online content, making the research process more manageable and less overwhelming.
```

Comment:
You might run into input problems with the ChatGPT API if you have articles above the character limit. If you end up addressing this edge case, you may want to looking into chunking the article by paragraph (to make sure the summaries are accurate and preserve the meaning of the original article) and sending multiple requestst to the API.